extensions/http/jetty/src/main/java/org/eclipse/dataspaceconnector/extension/jetty/JettyExtension.java

```diff
@@ -28,6 +36,10 @@ public class JettyExtension implements ServiceExtension {
     private static final String KEYSTORE_PASSWORD = "keystore.password";
     @EdcSetting
     private static final String KEYMANAGER_PASSWORD = "keymanager.password";
+    @EdcSetting
```

Comment:
Possible to prefix the previous settings with edc.* ?

Reply:
makes sense. prefixed with `edc.web.https.`